composer.lock

```diff
@@ -8380,63 +8380,6 @@
             "abandoned": "scheb/2fa-bundle",
             "time": "2020-10-30T19:24:18+00:00"
         },
-        {
-            "name": "sensio/distribution-bundle",
```

Comment:
it seems composer.lock was not properly cleaned in #5761... mea culpa!
@Kdecherf @j0k3r @nicosomb I just opened #5839 so that the fix is not mixed with this PR.
I'll rebase this PR on it